docs/variants/zir_blazer/hardware_matrix.md

```diff
@@ -0,0 +1,52 @@
+# Hardware configuration matrix
+
+The tables below contain information about current components of
+workstations for Zir_Blazer available in 3mdeb testing laboratory.
+
+## LA1E312159
+
+| Component                      | Description                                      |
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel Core i5-12600K 3.7                         |
+|                                | Noctua NH-U12S CPU Cooler                        |
+| **GPU**                        | N/A                                              |
+| **RAM**                        | Slot 1: Kingston KF436C17BBK4/32                 |
+|                                | Slot 2: Kingston KF436C17BBK4/32                 |
+|                                | Slot 3: Kingston KF436C17BBK4/32                 |
+|                                | Slot 4: Kingston KF436C17BBK4/32                 |
+| **SSD**                        | Intel 670p 512 GB M26472-201 NVME                |
+| **Flash memory**               | TBD                                              |
```

Comment:
Information about flash memory should be added

Reply:
added